tests/Processor/DateProcessorTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+
+namespace RenanBr\BibTexParser\Test\Processor;
+
+
+use DateTime;
+use DateTimeInterface;
+use DateTimeZone;
+use PHPUnit\Framework\TestCase;
+use RenanBr\BibTexParser\Processor\DateProcessor;
+
+class DateProcessorTest extends TestCase
```

Comment:
Could you add tests for these cases that processor should bypass?

```
['month' => 'Jul', 'year' => '2000']; // missing day in month
['month' => '05', 'year'=> '2000'];   // invalid month value
['month' => '1~jan', 'year' => '98']; // invalid year
['month' => 'foo', 'year' => 'bar'];  // invalid
['month' => '1~jan'];                 // missing year
['year' => '2000'];                   // missing month
[];                                   // missing everything 
['_date' => 'I do exist'];            // already existing _date
```

Reply:
bypass or exception ?

Reply:
bypass, as you did :smile: